pipeline.go

```diff
@@ -174,12 +174,19 @@ func (r *PipelineDefinition) UnmarshalJSON(data []byte) error {
 	}
 	result.Version = parsedJson.Version
 	for name, service := range parsedJson.Services {
+		service.Meta = ServiceMeta{
+			Type: ServiceTypeService,
+		}
 		result.Steps[name] = service
 	}
 	for name, step := range parsedJson.Steps {
 		if _, found := result.Steps[name]; found {
 			return fmt.Errorf("Duplicate step/service '%s'", name)
 		}
+		step.Meta = ServiceMeta{
+			Type:      ServiceTypeStep,
+			KeepAlive: KeepAliveNo,
+		}
```

Comment:
This code doesn't seem to match the commit title?

Reply:
The commit title states the effect, this is the needed change. Can be renamed, I'm open for suggestions.